### PR TITLE
fix(slider): add explicit labeling for slider ranges and input

### LINKED ISSUE #1364: ♿Slider AVT1 - August 2018♿
## Environment
Chrome Version 70.0.3538.67 (Official Build) (64-bit)
macOS Mojave 10/Windows 10

## Violations:
Each form control must have associated label.
screen shot 2018-10-30 at 2 03 20 am

The id slider referenced by the 'for' attribute of a label element is not a valid form input element or an element with id slider is either null or does not exist.
screen shot 2018-10-30 at 2 04 35 am

## How to reproduce
1. [Download the Dynamic Accessibility Plugin tool here](https://ibm.ent.box.com/notes/335005856253?s=mhpgjlgss354mcidsv6rvpuu0bxlmfm4)
2. Run the extension using the August 2018 ruleset

### PR DESCRIPTION
**Pull request by dakahn**
Closes #1364 

Fixes up some labelling issues that were making DAP ornery. Adds explicit labels to each range and adds an aria-label attribute to the actual input.

### CODE REVIEW

**Review by asudoh — COMMENTED** [review]
LGTM basically, it'd be great if you can add a mechanism to avoid ID duplication like the old `<input>` code does. Thanks @dakahn!

**Comment by dakahn** [comment]
Cool pushed up a change incorporating the input ID into the IDs of the labels.

**Review by tw15egan — APPROVED** [review]
Awesome! Thanks for tackling all these a11y issues. Great work 💪

File: src/components/slider/slider.hbs [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
       <div class="bx--slider__filled-track"></div>
-      <input id="slider" class="bx--slider__input" type="range" step="1" min="0" max="100" value="0">
+      <input aria-label="slider" id="slider" class="bx--slider__input" type="range" step="1" min="0" max="100" value="0">
     </div>
-    <span class="bx--slider__range-label">0</span>
-    <span class="bx--slider__range-label">100</span>
-    <input id="{{inputId}}" type="number" class="bx--text-input bx--slider-text-input{{#if light}} bx--text-input--light{{/if}}" placeholder="0">
+    <label id="{{inputId}}_bottom-range-label" class="bx--slider__range-label">0</label>
+    <label id="{{inputId}}_top-range-label" class="bx--slider__range-label">100</label>
+    <input id="{{inputId}}" aria-labelledby="{{inputId}}_bottom-range-label {{InputId}}_top-range-label" type="number" class="bx--text-input bx--slider-text-input{{#if light}} bx--text-input--light{{/if}}" placeholder="0">
```
Comment (asudoh) [inline src/components/slider/slider.hbs]:
```suggestion
    <input id="{{inputId}}" aria-labelledby="{{inputId}}_bottom-range-label {{inputId}}_top-range-label" type="number" class="bx--text-input bx--slider-text-input{{#if light}} bx--text-input--light{{/if}}" placeholder="0">
```

**Review by asudoh — APPROVED** [review]
Thanks @dakahn for your quick change! Once the character case thing is solved the code is good to go.

File: src/components/slider/slider.hbs (hunk 1 above)
Comment (alisonjoseph) [reply src/components/slider/slider.hbs]: @dakahn

**Comment by dakahn** [comment]
@asudoh @alisonjoseph Sorry I slept on this for so long.

**Pull request merged**